webpack/ForemanTasks/Components/TaskDetails/Components/__tests__/__snapshots__/TaskInfo.test.js.snap

```diff
@@ -32,7 +32,9 @@ exports[`TaskInfo rendering render with Props 1`] = `
       sm={6}
     >
       <span>
-        Monitor Event Queue
+        <EllipisWithTooltip>
```

Comment:
Wonder why `EllipsisWithTooltip` generates `EllipisWithTooltip` in the snapshot :thinking: